src/style.css

```diff
@@ -1,4 +1,63 @@
-.hello {
-  color: red;
-  background: url('./img.png');
+/* stylelint-disable */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: 'Open Sans', 'Lucida Grande', tahoma, verdana, arial, sans-serif;
+  color: #545862;
+  font-weight: 400;
+  border: none;
+  font-size: calc(1.5svh + 1px);
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100svh;
+  background-color: #545862;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=CSS%20Loader.-,Your%20list%20should%20be%20a%20clone%20of%20the%20part%20of%20the%20minimalist%20project%20captured%20in%20the%20video%20below,-.), please add a proper styling to your app to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/EvansSnave/to-do-list/assets/68796232/ecb08986-239d-4370-9f73-ae15aa8ecbc5)  | ![image](https://github.com/EvansSnave/to-do-list/assets/68796232/cd3b58a3-9f86-456b-bd3a-d74e1ee92773) |

Reply:
That's weird. For me it has all the styling look 
![image](https://github.com/EvansSnave/to-do-list/assets/127534196/8a425930-d641-4de1-88f7-6d9ebd5c58c9)

Reply:
@Omar-Muhamad I can see it when I go live in Vs code